### PR TITLE
[PAY-972] Handle duplicate wallet connections properly

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

We would only show the error if you'd previously connected a wallet, on the `WalletsTable`
Only shown here previously:
Screen Shot 2023-02-28 at 3 34 14 PM

Now also showing here:

Screen Shot 2023-02-28 at 3 41 41 PM

### Dragons

[] Need to make sure this works on mobile
[] Doublechecking that this text goes away when a wallet *is* successfully connected

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/piazz-wallet-duplicates

File: packages/web/src/pages/audio-rewards-page/components/WalletsTable.tsx [hunk 1]
```diff
@@ -251,7 +253,9 @@ const WalletsTable = ({
           isConfirmRemoving={false}
         />
       )}
-      {errorMessage && <div className={styles.error}>{errorMessage}</div>}
+      {!suppressError && errorMessage ? (
```
Comment (dylanjeffers) [inline packages/web/src/pages/audio-rewards-page/components/WalletsTable.tsx:256]: should we just remove this here, or is it needed for another component usage?

**Review by dylanjeffers — APPROVED** [review]
make sense, just wondering about if we need the duplicate or not

File: packages/web/src/pages/audio-rewards-page/components/WalletsTable.tsx (hunk 1 above)
Comment (piazzatron) [reply packages/web/src/pages/audio-rewards-page/components/WalletsTable.tsx:256]: Ah I think you're right and we can remove it actually. It's used in the AudioBreakdownModal but don't think we care

Comment (piazzatron) [reply packages/web/src/pages/audio-rewards-page/components/WalletsTable.tsx]: fixed. good call

**Pull request merged**